### PR TITLE
Add GitHub Teacher Bio

### PR DESCRIPTION
**Pull request by githubteacher**
This Pull Request adds a bio for @githubteacher as requested in #30.

@jaw6 what do you think of these changes?

### CODE REVIEW

**Comment by jaw6** [comment]
> what do you think of these changes? 

Looks like you pulled in @sf122240's bio also -- intentional or ?

**Comment by githubteacher** [comment]
This was unintentional, @jaw6.  Maybe I should clean this up before we merge.  Thanks for catching that! :+1:

File: bios/githubteacher.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## GitHub Teacher
+
+**Name:** GitHub Teacher
+
+**Occupation:** Teacher
+
+**Location:** San Francisco
```
Comment (githubteacher) [inline bios/githubteacher.md]: Is this the one in California?  Maybe we should be specific.

**Comment by githubteacher** [comment]
:+1: This looks good for now.

**Pull request merged**